bin/create_folders.py

```diff
@@ -226,11 +264,14 @@ def create_folders(
     project_name = sys.argv[1]
     submission_id = sys.argv[2]
     create_or_update = sys.argv[3]
-    predictions_file = sys.argv[4] if len(sys.argv) > 4 else None
+    # TODO: https://sagebionetworks.jira.com/browse/ORCA-311
+    predictions_file = sys.argv[4] if len(sys.argv) == 5 else None
```

Comment:
Why do you have to do this? aren't there only 3 hard-coded arguments for `CREATE_FOLDERS`?

Reply:
Oh I see, this is to allow this script to be used in `run_docker.py`?

Reply:
Correct